OptimizelySDK/OptimizelyDecisions/OptimizelyDecision.cs

```diff
@@ -20,23 +20,33 @@
 
 namespace OptimizelySDK.OptimizelyDecisions
 {
+    /// <summary>
+    /// OptimizelyDecision defines the decision returned by decide api.
+    /// </summary>
     public class OptimizelyDecision
     {
+        // variation key for optimizely decision.
```

Comment:
this documentation is wrong, please use `///` it will generate template and put information over there.